src/python/pants/backend/python/tasks/pytest_run.py

```diff
@@ -112,6 +112,13 @@ def register_options(cls, register):
              help='Emit coverage information for specified packages or directories (absolute or '
                   'relative to the build root).  The special value "auto" indicates that Pants '
                   'should attempt to deduce which packages to emit coverage for.')
+    # TODO: The default is False for backwards compatiblity. We almost certainly want to change
+    # that after a deprecation cycle.
+    register('--coverage-omit-test-sources', fingerprint=True, type=bool, default=False,
+             help='Whether to omit test source files from coverage reports.')
+    register('--coverage-reports', fingerprint=True, choices=('xml', 'html'), type=list,
+             member_type=str, default=('xml', 'html'),
```

Comment:
Does the default mean to emit both? If so, why is this a good default?

Reply:
Because it's what we were already doing prior to this change.